lib/idStorage.js

```diff
@@ -0,0 +1,61 @@
+const storage = new Map();
```

Comment:
It will yes, but that'd the least of our problems, lots of things are being used with no support below IE11. Attrobj being the first one which is deep into Domodule. Can use a regular object if you'd still like to strike for IE10 support but this is more performant for the use we are doing.